spec/models/hearing_spec.rb

```diff
@@ -283,4 +283,12 @@
       expect(hearing.hearing_day_regional_office).to eq(nil)
     end
   end
+
+  context "aod" do
+    let!(:hearing) { create(:hearing) }
```

Comment:
```suggestion
    let(:unrecognized_appellant) { create(:claimant, type: "OtherClaimant") }
    let(:appeal) { create(:appeal, claimants: [unrecognized_appellant]) }
    let!(:hearing) { create(:hearing, appeal: appeal) }
```
This will cause `hearing.appeal.appellant.unrecognized_claimant?` to evaluate to `true` in the test.